tests/countries/test_dr_congo.py

```diff
@@ -0,0 +1,189 @@
+#  holidays
+#  --------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see CONTRIBUTORS file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+
+from unittest import TestCase
+
+from holidays.countries.dr_congo import (
+    DRCongo,
+    CD,
+    COD,
+)
+from tests.common import CommonCountryTests
+
+
+class TestDRCongo(CommonCountryTests, TestCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass(DRCongo, years=range(1980, 2050))
+
+    def test_country_aliases(self):
+        self.assertAliases(DRCongo, CD, COD)
+
+    def test_no_holidays(self):
+        self.assertNoHolidays(DRCongo(years=1979))
+
```

Comment:
It's too early to talk about tests when the holidays are not yet finalized, but I don't see our usual set of methods for the holidays here. 🙄

Reply:
Surely, the tests should follow recent standards.